README.adoc

```diff
@@ -24,6 +24,37 @@ See the official documentation on https://docs.teragrep.com[docs.teragrep.com].
 . Setup database according to end-user needs
 . Can be deployed using https://tomcat.apache.org/tomcat-9.0-doc/deployer-howto.html[tomcat]/JBoss
 
+== How to run in a container?
+
+Containerized deployment for CFE-18. Requires external database to use. It uses public tomcat docker image (tomcat:8-jre8-temurin-focal)
+
+Expected files are:
```

Comment:
Expected files are actually
/etc/tomcat/Catalina/localhost/cfe-18/application.properties
/etc/tomcat/Catalina/localhost/cfe-18/log4j2.xml

and example files are  the docker/... ones